Strategy11/formidable-forms
# Fix Footer Misplacement on Initial Page View

This PR addresses the issue of the footer being misplaced at the bottom when first viewing a page. The footer will now appear in its correct position without requiring the user to scroll.

## Related Issue:
[Footer Misplacement on Initial Page View](https://github.com/Strategy11/formidable-pro/issues/4508)

## QA URL
https://qa.formidableforms.com/sherv2/wp-admin/admin.php?page=formidable-entries&frm_action=new

## Testing Instruction:
1. Navigate to `WP Admin > admin.php?page=formidable-entries&frm_action=new`.
2. Verify the footer is now in its correct position on initial page view, without needing to scroll down.

## Output Comparision:
### Before
![2023-10-22 13 12 39](https://github.com/Strategy11/formidable-forms/assets/69119241/3d9a6049-689c-434a-927f-323564975155)

### After
<img width="1078" alt="image" src="https://github.com/Strategy11/formidable-forms/assets/69119241/d5d59ddd-0c86-45e3-9d38-9716764cf5e6">
